### PR TITLE
i fixed it

### PR DESCRIPTION
**Pull request by 00p513-dev**

### CODE REVIEW

**Review by Gangster8192 — APPROVED** [review]

**Review by Timdpr — APPROVED** [review]

**Review by AAGaming00 — APPROVED** [review]

**Review by botatooo — APPROVED** [review]

**Review by csskroubledev — APPROVED** [review]

**Review by TheOddGarlic — APPROVED** [review]
Approved.

**Review by Abhi011999 — APPROVED** [review]

**Review by csskroubledev — APPROVED** [review]

**Review by fracturedface — APPROVED** [review]

**Review by Zorotic — APPROVED** [review]

**Review by AGuyNamedJens — APPROVED** [review]

**Review by Sven65 — APPROVED** [review]
LGTM.

**Review by thetayloredman — APPROVED** [review]

**Review by KryskZ09 — APPROVED** [review]

**Review by Hwiggy — APPROVED** [review]
Looks good to me

**Comment by thetayloredman** [comment]
You are legally required to merge this.

File: 2020/10/2020-10-23-RIAA.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ GitHub
 
 Dear Sir or Madam:
 
-I am contacting you on behalf of the Recording Industry Association of America, Inc. (RIAA) and
-its member record companies. The RIAA is a trade association whose member companies
+I am contacting you on behalf of the Recording Industry Association of America, Inc. (a bunch of cunts who abuse monopolies) and
+its member record companies. The RIAA is a organistaion full of dumbfucks whose member companies
 create, manufacture or distribute sound recordings representing approximately eighty-five (85)
-percent of all legitimate recorded music consumption in the United States. Under penalty of
+percent of all legitimate recorded music consumption in the United States, simply we have a monopoly and we take down anyone we dont like. Under penalty of
```
Comment (thetayloredman) [inline 2020/10/2020-10-23-RIAA.md]:
```suggestion
percent of all legitimate recorded music consumption in the United States, simply we have a monopoly and we take down anyone we don't like. Under penalty of
```

File: 2020/10/2020-10-23-RIAA.md [hunk 2]
```diff
@@ -4,10 +4,10 @@ GitHub
...
 perjury, we submit that the RIAA is authorized to act on behalf of its member companies on
```
Comment (thetayloredman) [inline 2020/10/2020-10-23-RIAA.md]:
```suggestion
perjury, we submit that the RIAA is not authorized to act on behalf of its member companies on
```

**Review by thetayloredman — CHANGES_REQUESTED** [review]

**Review by A7U — APPROVED** [review]
Needs some more insults but it's pretty good so far.

**Review by Favna — APPROVED** [review]

**Comment by thetayloredman** [comment]
True.

Feel free to suggest changes with insults.

```js
RIAA.insults++
```

**Review by awnumar — APPROVED** [review]

**Review by AGuyNamedJens — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md [hunk 3]
```diff
@@ -4,10 +4,10 @@ GitHub
 
 Dear Sir or Madam:
 
-I am contacting you on behalf of the Recording Industry Association of America, Inc. (RIAA) and
-its member record companies. The RIAA is a trade association whose member companies
+I am contacting you on behalf of the Recording Industry Association of America, Inc. (a bunch of cunts who abuse monopolies) and
+its member record companies. The RIAA is a organistaion full of dumbfucks whose member companies
```
Comment (thetayloredman) [inline 2020/10/2020-10-23-RIAA.md:8]:
```suggestion
its member record companies. The RIAA is a organization full of dumbfucks whose member companies
```

It's a legal notice, we have to spell right!

File: 2020/10/2020-10-23-RIAA.md (hunk 1 above)
Comment (thetayloredman) [inline 2020/10/2020-10-23-RIAA.md]:
```suggestion
percent of all legitimate (and illegitimate) recorded music consumption in the United States, simply we have a monopoly and we take down anyone we dont like. Under penalty of
```

Be honest...

File: 2020/10/2020-10-23-RIAA.md [hunk 4]
```diff
@@ -37,69 +37,7 @@ https://github.com/tejaskhot/youtube-dl
...
-
-Indeed, the comments in the youtube-dl source code make clear that the source code was
-designed and is marketed for the purpose of circumventing YouTube’s technological measures
-to enable unauthorized access to our member’s copyrighted works, and to make unauthorized
-copies and distributions thereof: they identify our member’s works, they note that the works
-are VEVO videos (virtually all of which are owned by our member companies), they
-acknowledge the those works are licensed to YouTube under the YouTube standard license, and
-they use those examples in the source code to describe how to obtain unauthorized access to
-copies of our members’ works.
+The clear purpose of this source code is to (i) download porn and (ii) download porn
```
Comment (thetayloredman) [inline 2020/10/2020-10-23-RIAA.md]:
```suggestion
The clear purpose of this source code is to (i) download porn, (ii) download porn, and (iii) download porn
```

File: 2020/10/2020-10-23-RIAA.md [hunk 5]
```diff
@@ -110,9 +48,9 @@ This e-mail does not constitute a waiver of any right to recover damages incurre
 any such unauthorized activities, and all such rights as well as claims for other relief are
 expressly reserved.
 
-You may contact me at RIAA, [private] Tel. [private],  
-or email [private] to discuss this notice.
+You may contact me at RIAA, a massive cock Tel. +69 42080085,  
+or email wehatefoss@riaa.com to discuss this notice.
 
 Sincerely,
```
Comment (thetayloredman) [inline 2020/10/2020-10-23-RIAA.md]:
```suggestion
Not very sincerely,  
```

**Review by alcalyn — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md [hunk 6]
```diff
@@ -110,9 +48,9 @@ This e-mail does not constitute a waiver of any right to recover damages incurre
...
-You may contact me at RIAA, [private] Tel. [private],  
-or email [private] to discuss this notice.
+You may contact me at RIAA, a massive cock Tel. +69 42080085,  
+or email wehatefoss@riaa.com to discuss this notice.
 
 Sincerely,  
-[private]  
-[private]
+Wayne Kerr
+A huge cunt
```
Comment (thetayloredman) [inline 2020/10/2020-10-23-RIAA.md:56]:
```suggestion
A huge cunt
RIAA Cunts Department
```

File: 2020/10/2020-10-23-RIAA.md (hunk 3 above)
Comment (byte-chan) [inline 2020/10/2020-10-23-RIAA.md:8]:
```suggestion
its member record companies. The RIAA is an organisation full of dumbfucks whose member companies
```

**Review by thetayloredman — CHANGES_REQUESTED** [review]
Few things to change

**Review by MultHub — CHANGES_REQUESTED** [review]

File: 2020/10/2020-10-23-RIAA.md (hunk 3 above)
Comment (thetayloredman) [reply 2020/10/2020-10-23-RIAA.md:8]: Already suggested but mk.

**Comment by thetayloredman** [comment]
Frick you, RIAA! You really should stop digging in the mud and playing with about ALL **OF THE DISCORD MUSIC BOTS**

**Review by AGuyNamedJens — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md (hunk 1 above)
Comment (mathisve) [reply 2020/10/2020-10-23-RIAA.md]: Good catch!

**Review by Mathisco-01 — APPROVED** [review]

**Review by Thewsomeguy — APPROVED** [review]

**Review by tommywxd — APPROVED** [review]
looks pretty good to me

**Review by PauloGoncalvesLima — APPROVED** [review]
Approved

**Review by imachug — APPROVED** [review]

**Review by OkameProductions — APPROVED** [review]

**Review by d4n3436 — APPROVED** [review]

**Comment by thetayloredman** [comment]
Prediction:

```
@github closed this 1 minute ago
```

heh

**Review by dequeues — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md (hunk 3 above)
Comment (s0urfruit) [reply 2020/10/2020-10-23-RIAA.md:8]: I mean s or z changes wether its British English or american English

**Review by TorchedSammy — APPROVED** [review]

**Review by tommywxd — APPROVED** [review]
Approved.

**Review by TheRandomMelon — APPROVED** [review]
Best change ever made

**Review by btwiusegentoo — APPROVED** [review]
Yes

**Review by alenpaul2001 — APPROVED** [review]

**Review by hack-everything — APPROVED** [review]

**Review by moom0o — APPROVED** [review]

**Review by mininmobile — APPROVED** [review]
image

**Review by xKiraiChan — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md (hunk 3 above)
Comment (00p513-dev) [reply 2020/10/2020-10-23-RIAA.md:8]: i bri ish innit

**Review by Chrismettal — APPROVED** [review]
Ready to merge.

**Review by lululombard — APPROVED** [review]

**Review by botatooo — APPROVED** [review]

**Review by DortDev — APPROVED** [review]
this is golden.

**Review by eatradish — APPROVED** [review]

**Comment by 00p513-dev** [comment]
@github merge wen

**Review by woina — APPROVED** [review]

**Review by advilm — APPROVED** [review]

**Review by PantheraRed — APPROVED** [review]

**Comment by TheAwesome98-Real** [comment]
You sure fixed it, amy

**Review by analotia — APPROVED** [review]

**Review by dicedtomatoreal — APPROVED** [review]

**Review by GottZ — APPROVED** [review]
i wonder if reviewing this can get me banned or let me get on a "what's the word again"-list. let's see..

free speech on the internet is certainly not liked in 2020 it seems.

**Comment by 00p513-dev** [comment]
> i wonder if reviewing this can get me banned or let me get on a "what's the word again"-list. let's see..
> 
> free speech on the internet is certainly not liked in 2020 it seems.

hopefully you will be ok

**Review by csskroubledev — APPROVED** [review]

**Review by Nefomemes — APPROVED** [review]

**Review by alenpaul2001 — APPROVED** [review]

**Comment by Sven65** [comment]
> i wonder if reviewing this can get me banned or let me get on a "what's the word again"-list. let's see..
> 
> free speech on the internet is certainly not liked in 2020 it seems.

Hey, I know you!

What's this about a ban lmao?

**Comment by 00p513-dev** [comment]
@github merge wen

**Comment by TheAwesome98-Real** [comment]
> @github merge wen

eta s0n hopefully

**Review by Binkus — APPROVED** [review]
Looks 100% fine.

**Comment by jessephus** [comment]
Hi :wave: We are closing this pull request because we [do not accept contributions](https://github.com/github/dmca/blob/master/CONTRIBUTING.md) to this repository. Feel free to keep the discussion going in this thread. Please just be mindful to keep the discussion respectful and consistent with [GitHub’s Community Guidelines](https://docs.github.com/github/site-policy/github-community-guidelines).

We understand that you may have questions about this takedown. Although we do not normally monitor comments posted to this repository, our team is reading through the comments and questions in this and other recent threads to see where we may be able to clarify our policies or answer some common questions. Thanks.

**Comment by thetayloredman** [comment]
@jessephus [r/whooosh](https://reddit.com/r/whooosh/) moment

**Comment by TheAwesome98-Real** [comment]
> @jessephus [r/whooosh](https://reddit.com/r/whooosh/) moment

think it's an autoclose response from a dev's acc

**Comment by thetayloredman** [comment]
tbh probably wouldn't be surprised

but why wouldn't it have fired a while ago?

**Comment by jessephus** [comment]
I am a real person. It was not an auto-close response. And I totally understand joke/satire here. I wish I could say more or join in on the fun, but since this is about a pending legal matter... 🙊

**Comment by thetayloredman** [comment]
> I am a real person.

*So you're human? **Select all images with traffic lights.*** (ok fine)

> but since this is about a pending legal matter...

A legal matter that broke almost half of discord music bots? 🙊

**Comment by Nytelife26** [comment]
@jessephus You want to point out to me which part of this is a joke or satire? Trust me when I say all of us are ABSOLUTELY, GENUINELY, 100% serious. Every single one of us. This is a petty move by the RIAA and it shouldn't fly in any room on earth, let alone in court against one of the biggest and most supportive FOSS organizations.

Pending legal matter my ass. Tell them to grow a pair of nuts and stop taking away valid automation tools that are useful to a HUGE amount of people over their "cOpYRiGhT" for which they are claiming this based on THREE videos out of YouTube's whole platform.

Sincerely, everyone that's ever used YTDL, and everyone that dislikes huge corporations sticking their thumbs in people's asses just to be an inconvenience over things that don't matter or hurt them in the slightest.

**Comment by 00p513-dev** [comment]
I regularly used youtube dl for things other than youtube too. such as i mentioned in my edit...

**Comment by Dev-i-l** [comment]
i support the cause

**Comment by TheAwesome98-Real** [comment]
This is like satire’s serious cousin

**Comment by Abhi011999** [comment]
No worry guys as we know that the repo's back now and the conflicting code is removed/modified.

**Pull request closed without merging**